### PR TITLE
Basic Search

### PR DESCRIPTION
**Pull request by elland**
🕵️‍♀️ 

[Preview 1](https://cloud.githubusercontent.com/assets/54423/19111043/c05f7cf2-8afd-11e6-926e-0db5273e42a9.png).
[Preview 2](https://cloud.githubusercontent.com/assets/54423/19111044/c0604ac4-8afd-11e6-80bc-077d147dc82a.png).

### CODE REVIEW

**Comment by elland** [comment]
@3lvis give it a go 💃

**Comment by 3lvis** [comment]
How does it look!?

File: iOS/Source/BookmarksController.swift [hunk 1]
```diff
@@ -117,7 +144,10 @@ extension BookmarksController: UITableViewDelegate {
         }
 
         let editRowAction = UITableViewRowAction(style: .normal, title: "Edit") { (action, indexPath) in
-            // edit and update bookmark
+            guard let path = self.bookmarksDataSource.item(at: indexPath).path else { fatalError("!") }
+            let bookmarkController = BookmarkController(path: path, fetcher: self.fetcher, delegate: self)
```
Comment (3lvis) [inline iOS/Source/BookmarksController.swift]: Is this branch outdated? I think I commented that we should send the object here.

Comment (elland) [reply iOS/Source/BookmarksController.swift]: Looks like I forgot to rebase it.

Comment (elland) [reply iOS/Source/BookmarksController.swift]: Check now.

**Comment by elland** [comment]
@3lvis ? 👍 ship it?

File: iOS/Source/BookmarkSearchResultsController.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        super.viewDidLoad()
+
+        self.tableView.dataSource = self
+        self.tableView.delegate = self
+        self.tableView.register(BookmarkCell.self)
+    }
+
+    init(delegate: BookmarkSearchResultsControllerDelegate? = nil) {
+        super.init(with: .plain)
+        self.delegate = delegate
```
Comment (3lvis) [inline iOS/Source/BookmarkSearchResultsController.swift:27]: Newline after super call, plz.

Comment (elland) [reply iOS/Source/BookmarkSearchResultsController.swift:27]: 🆗

**Comment by 3lvis** [comment]
SHIP IT!

**Pull request merged**